qml/TabLayout.qml

```diff
@@ -206,6 +214,7 @@ Item {
             // view with the different views available in a tab
             StackView {
                 id: stack
+                property int stack_id: 0
                 anchors.fill: parent
```

Comment:
In `stack_component.createObject()` calls anchors are set. However, here they are already set by default. If arguments are deleted, the following error arises: 
`QML StackView: Detected anchors on an item that is managed by a layout. This is undefined behavior; use Layout.alignment instead.`
Tabs layout is handled correctly if no anchors are set.